groups/groups.yaml

```diff
@@ -1198,6 +1198,17 @@ groups:
       - nikitaraghunath@gmail.com
       - stefan.schimanski@gmail.com
 
+  - email-id: k8s-infra-staging-scheduler-plugins@kubernetes.io
+    name: k8s-infra-staging-scheduler-plugins
+    description: |-
+      ACL for staging scheduler-plugins
+    settings:
+      ReconcileMembers: "true"
+    members:
+      - ahg@google.com
+      - wangqingcan1990@gmail.com
+      - wei.huang1@ibm.com
```

Comment:
@Huang-Wei I believe the email addresses listed here need to be tied to a GSuite account(i.e. GMail). Please confirm this email is tied to a GSuite account.

Reply:
@seanmalloy Please update mine to `hweicdl@gmail.com`.

Reply:
@Huang-Wei fixed.